riscv-executor/src/lib.rs

```diff
@@ -2854,6 +2877,7 @@ pub fn execute<F: FieldElement>(
     prover_ctx: &Callback<F>,
     bootloader_inputs: &[F],
     max_steps_to_execute: Option<usize>,
+    full_witness: bool,
```

Comment:
Shouldn't this just be ExecMode?

Reply:
but then you could also pass `Fast`, which is invalid

Reply:
exec mode is internal to the crate

Reply:
yea, just don't really like `bool` parameters..but ok